spec/support/factory_girl.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.configure do |config|
+  config.include FactoryGirl::Syntax::Methods
```

Comment:
This allows us to use `create` instead of `FactoryGirl.create` in specs. That will help when we upgrade at some point and have to change all the `FactoryGirl` to `FactoryBot`.